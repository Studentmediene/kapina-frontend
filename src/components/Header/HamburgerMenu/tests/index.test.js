import React from 'react';
import { shallow } from 'enzyme';

import { HamburgerMenu } from '../';

describe('<HamburgerMenu />', () => {
  it('renders correctly', () => {
    const mockProps = {
      togglePlayPause: jest.fn(),
      paused: true,
    };
    const tree = shallow(<HamburgerMenu {...mockProps} />);
    expect(tree).toMatchSnapshot();
  });
});
