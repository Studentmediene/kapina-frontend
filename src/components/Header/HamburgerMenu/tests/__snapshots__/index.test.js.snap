// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HamburgerMenu /> renders correctly 1`] = `
<div
  className="container"
  onClick={[Function]}
>
  <img
    className="hamburgerIcon"
    src="hamburger-menu.svg"
  />
  <div
    className="hamburgerText"
  >
    Meny
  </div>
  <div
    className="overlay"
  />
  <withRouter(NavDrawer)
    links={
      Array [
        Object {
          "path": "/",
          "title": "Forside",
        },
        Object {
          "path": "/programmer",
          "title": "Programmer",
        },
        Object {
          "path": "/sendeplan",
          "title": "Sendeplan",
        },
      ]
    }
    onNavigation={[Function]}
    open={false}
  />
</div>
`;
